llvm/test/tools/llvm-mca/RISCV/SiFive7/strided-load-store.s

```diff
@@ -0,0 +1,361 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
```

Comment:
I am kind of confused on how these MCA tests are organized.

We have `llvm/test/tools/llvm-mca/RISCV/vle-vse.s`, but why is this `strided-load-store.s` test put under `lvm/test/tools/llvm-mca/RISCV/SiFive7`? They should test similar things I think.

If we want to put CPU (or schedule model) specific tests in its own directory, then we should move `llvm/test/tools/llvm-mca/RISCV/*` to `lvm/test/tools/llvm-mca/RISCV/SiFive7`?

Reply:
I think it would make most sense to have a directory for each schedule model. What do you think about me fixing that in a follow up patch, as the test here is in the correct directory already?

Reply:
Yes, please go ahead!